core/src/test/scala/org/apache/spark/ui/UISuite.scala

```diff
@@ -179,16 +179,15 @@ class UISuite extends SparkFunSuite {
   test("verify appUIAddress contains the scheme") {
```

Comment:
You might also change occurrences of "appUIAddress" in places like this test name. Have a look with a search and replace.